### ISSUE TITLE
step2_SPAtests.R ignores --vcfFileIndex in v0.44.6.5

### ISSUE THREAD

**Issue by nkgx** [weizhouUMICH/SAIGE#370]
I am running SAIGE with a VCF + TBI index (not CSI) by providing a --vcfFileIndex=abc.vcf.gz.tbi parameter. This used to work in older versions but in v0.44.6.5 I am getting this error:

```
Error in SPAGMMATtest(vcfFile = opt$vcfFile, vcfFileIndex = opt$vcfFileIndex,  : 
  ERROR! vcfFileIndex abc.vcf.gz.csi does not exist
```

I believe this is caused by these lines, which will always look for a vcf.gz.csi file even if --vcfFileIndex is provided: https://github.com/weizhouUMICH/SAIGE/blob/57c43dbc6622a154f434fcf696e767b1820f1e7a/R/SAIGE_SPATest.R#L294

Would it be possible to fix this?

**Comment by mtdrlv** [comment]
I just stumbled upon the same error with v.44.6.5. 
Which version is safe to use with vcf +tbi files until this problem is fixed?

**Comment by ytao0210** [comment]
I met the same problem. How did you solve this?

**Comment by ytao0210** [comment]
You can use "tabix -C *.vcf.gz" to generate a csi index.

**Comment by mtdrlv** [comment]
I generated csi index files and these worked OK

**Comment by weizhouUMICH** [comment]
Sorry for the late reply! We have just released a new version 1.0.0. It now only uses csi for VCF index. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
